website/docs/docs/build/custom-schemas.md

```diff
@@ -84,6 +84,10 @@ The following code represents the default macro's logic:
 {%- endmacro %}
 ```
 
+import WhitespaceControl from '/snippets/_whitespace-control.md';
+
+<WhitespaceControl/>
```

Comment:
suggest moving this to line 67 above. it feels a little lost here between a code block and an H2 heading.

check the other instances and their placements as well.